politeiawww/proposals.go

```diff
@@ -3336,3 +3336,37 @@ func (p *politeiawww) processVoteDetailsV2(token string) (*www2.VoteDetailsReply
 
 	return vdr, nil
 }
+
+func (p *politeiawww) processMessageProposer(mp www.MessageProposer, u *user.User) (*www.MessageProposerReply, error) {
```

Comment:
An email address is required at the moment, but there will be a time in the near future that it will be optional.  Can we go ahead and have this route check if the user has an email address set and return a user error if they don't.